create-collections.sh

```diff
@@ -0,0 +1,15 @@
+
+# create bucket
```

Comment:
It could be cool to do all this on travel-app startup in golang itself, hooking the gocb management APIs.

Reply:
Yeah, I like the idea, I think maybe we can add this in a different PR in future!